### PR TITLE
Verification and Operator Naming

### PR DESCRIPTION
**Pull request by edwardcallahan**
These were needed for operator-courier verify using the name `akka-cluster-operator`

### CODE REVIEW

**Comment by edwardcallahan** [comment]
Using https://github.com/libopenstorage/operator/tree/master/deploy as a rerence model as it appears to adhere to best practice.

File: LICENSE [hunk 1]
```diff
@@ -1,201 +1,15 @@
...
+This software is licensed under the Apache 2 license, quoted below.
 
-   1. Definitions.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License. You may obtain a copy of
+the License at
 
-      "License" shall mean the terms and conditions for use, reproduction,
-      and distribution as defined by Sections 1 through 9 of this document.
+    [http://www.apache.org/licenses/LICENSE-2.0]
```
Comment (marcoderama) [inline LICENSE]: Is there any concern that that link may not work at some point, thus making the LICENSE unreadable in full?

**Review by marcoderama — APPROVED** [review]
Approved with one little, ignorable, question.

File: LICENSE (hunk 1 above)
Comment (edwardcallahan) [reply LICENSE]: Good question.  Actually, yes this looks like the better example to follow:
https://github.com/lightbend/console-charts/blob/master/LICENSE
The Copyright is at the end.
I had seen multiple projects using https://github.com/lightbend/cinnamon-ui/blob/master/LICENSE, but the full license is better. Thanks.

**Comment by marcoderama** [comment]
And still approved

**Pull request merged**